scss/_button-group.scss

```diff
@@ -15,11 +15,16 @@
   > .btn-check:checked + .btn,
   > .btn-check:focus + .btn,
   > .btn:hover,
-  > .btn:focus,
   > .btn:active,
   > .btn.active {
     z-index: 1;
   }
+
+  // Keep focused button in front of hover and active button when both 
+  // buttons directly beside each other
```

Comment:
The comment is probably useless since we're in the button group component, isn't it?